apps/analog-app/src/app/pages/shipping/index.page.ts

```diff
@@ -1,22 +1,31 @@
-import { AsyncPipe, CurrencyPipe, NgForOf } from '@angular/common';
-import { Component, inject, OnInit } from '@angular/core';
+import { AsyncPipe, CurrencyPipe } from '@angular/common';
+import { Component, inject } from '@angular/core';
 
 import { Observable } from 'rxjs';
 import { CartService } from '../../cart.service';
 
 @Component({
   selector: 'app-shipping',
   standalone: true,
-  imports: [NgForOf, CurrencyPipe, AsyncPipe],
-  templateUrl: './shipping.html',
-  styleUrls: ['./shipping.scss'],
+  imports: [CurrencyPipe, AsyncPipe],
+  template: `
+    <h3>Shipping Prices</h3>
+    @for (shipping of shippingCosts | async; track $index) {
+    <div class="shipping-item">
+      <span>{{ shipping.type }}</span>
+      <span>{{ shipping.price | currency }}</span>
+    </div>
+    }
+  `,
+  styles: `
+  h3 {
+  border: solid 1px darkblue;
+  }
+  `,
```

Comment:
Hi, can you please run prettier on all these files?

Reply:
Looks like we are still using pre-3.0 prettier (2.8.3). We should take this up with a separate PR. Please ignore this.